SPECS/kernel-uvm/kernel-uvm.spec

```diff
@@ -87,6 +87,16 @@ ExclusiveArch:  x86_64 aarch64
 %description
 The kernel package contains the Linux kernel.
 
+%package devel
+Summary:        Lightweight kernel Devel package
+Group:          System Environment/Kernel
+Requires:       %{name} = %{version}-%{release}
+Requires:       gawk
+Requires:       python3
```

Comment:
Just a qq - can we verify that the devel package really requires gawk or python3? It's a devel package, not a OS service or so that may have package dependencies...

Reply:
This is the same pattern used in other kerenl specs but I've removed this dependency as it seems like it isn't necessary.